Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 51 setup redux

## Related ticket:
[Jira ticker SCRUM-51](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-51?atlOrigin=eyJpIjoiY2I0NWJhMDI5ZjQ1NDY5NDk3ODBlY2Y3MTI5ODZiNzIiLCJwIjoiaiJ9)

## In this PR:
- Setup redux
- Write slice and reducer to fetch all Json Api Links from Drupal

## Testing instruction
- Checkout branch
- Reinstall frontend dependencies: ```npm i```
- Serve the page
- Install Redux plugin (if not installed yet)
- Check the redux state. It should have a Drupal state that have everything fetched from Drupal
- If CORS issue showed up, install the CORS bypass plugins and try again

## State screenshot
<img width="820" alt="image" src="https://github.com/user-attachments/assets/7824c10a-1799-44d3-ae3a-34a03f9527f2">
